crates/y-sweet-core/src/api_types.rs

```diff
@@ -34,6 +34,19 @@ pub struct AuthDocRequest {
     pub user_id: Option<String>,
     #[serde(default)]
     pub metadata: HashMap<String, Value>,
+    #[serde(skip_serializing_if = "Option::is_none", rename = "validForSeconds")]
+    pub valid_for_seconds: Option<u64>,
```

Comment:
Even though this value doesn't exactly make sense in a Jamsocket context, I suppose we should at least allow it to be a part of the request body (and probably log if we see it, too)

Reply:
If we expose the same parameter in Plane, we could make it work there, too. Plane has an equivalent internally.

Reply:
https://linear.app/drifting/issue/DIS-2795/[platform]-allow-y-sweet-auth-doc-endpoint-to-take-valid-for-seconds